src/store.js

```diff
@@ -1,9 +1,29 @@
 import { createStore, applyMiddleware, compose } from 'redux'
 import thunk from 'redux-thunk'
-import root from './reducer.js'
+import createLogger from "redux-logger";
 
-const createStoreWithMiddleware = compose(applyMiddleware(thunk))(createStore)
+import rootReducer from './reducer.js'
 
-export default createStoreWithMiddleware(root)
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 
-// export default createStore(reducers);
+const configureStore = () => {
+  const middlewares = [thunk];
+
+  // Redux-Logger must go last in the middleware chain to log correctly.
+  // Only instrument in dev environment
+  if (process.env.NODE_ENV !== "production") {
+    middlewares.push(createLogger);
+  }
```

Comment:
@hydrosquall curious, what is your reasoning for including redux  logger too?  I usually only use redux-devtools, and look at it's actions feed for the information that I got from redux logger before.  🤔

Reply:
Hm, good question. Initially it was just something I'd reflexively included because the early projects I followed used it. Later on, I've found it useful to have some form of Redux introspection capabilities when debugging cross-browser issues in browsers without Redux devtools like IE / Safari. I'm not strongly attached to it though, and happy to drop it from the PR if it  unnecessarily complicates things.

Reply:
ah, good to understand.  let's add just Redux Devtools with this PR (drop redux-logger for now).  Once we have that and we get #88 in, this one should be good to go.

and yes, committing our bundle does introduce a little more dev friction.  thanks for bearing with it!